qa/suites/rados/ssh/start.yaml

```diff
@@ -1,3 +1,4 @@
 tasks:
 - install:
+    extra_packages: [ceph-daemon]
```

Comment:
should this be an override in packaged.yaml?

Reply:
pushed another patch to do this